DuckDuckGo/NetworkProtection/AppTargets/BothAppTargets/NetworkProtectionNavBarPopoverManager.swift

```diff
@@ -157,10 +164,36 @@ final class NetworkProtectionNavBarPopoverManager: NetPPopoverManager {
                 _ = try? await self?.vpnUninstaller.uninstall(removeSystemExtension: true)
             })
 
+            let featureFlagger = NSApp.delegateTyped.featureFlagger
+            let tipsFeatureFlagInitialValue = featureFlagger.isFeatureOn(.networkProtectionUserTips)
+            let tipsFeatureFlagPublisher: CurrentValuePublisher<Bool, Never>
+
+            if let overridesHandler = featureFlagger.localOverrides?.actionHandler as? FeatureFlagOverridesPublishingHandler<FeatureFlag> {
+
+                let featureFlagPublisher = overridesHandler.flagDidChangePublisher
+                    .filter { $0.0 == .networkProtectionUserTips }
+
+                tipsFeatureFlagPublisher = CurrentValuePublisher(
+                    initialValue: tipsFeatureFlagInitialValue,
+                    publisher: Just(tipsFeatureFlagInitialValue).eraseToAnyPublisher())
+            } else {
+                tipsFeatureFlagPublisher = CurrentValuePublisher(
+                    initialValue: tipsFeatureFlagInitialValue,
+                    publisher: Just(tipsFeatureFlagInitialValue).eraseToAnyPublisher())
+            }
```

Comment:
This is all to handle the feature flag.  The `CurrentValuePublisher` needs an initial value, and a publisher for subsequent values.

I think some of this can be simplified in the feature flag handling logic, as it's mostly boilerplate.  But here I wasn't too worried because FF code is temporary and this will go away anyway.

I can try to work on this if we see value, though.